### PR TITLE
Ltss enablement

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -331,4 +345,6 @@ jobs:
         env:
           CONTAINER_RUNTIME: ${{ matrix.container_runtime }}
           OS_VERSION: ${{ matrix.os_version }}
+          REGISTRY_LOGIN_USERNAME: ${{ secrets.REGISTRY_LOGIN_USERNAME }}
+          REGISTRY_LOGIN_PASSWORD: ${{ secrets.REGISTRY_LOGIN_PASSWORD }}  }}
```
Comment (dcermak) [inline .github/workflows/ci.yaml:349]:
```suggestion
```

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -319,6 +327,12 @@ jobs:
           -----END PGP PUBLIC KEY BLOCK-----
           EOF
 
+      - name: Login to the registry if needed
+        run: |
+          if [ "$TARGET" = "ibs-released" ]; then
+            echo $REGISTRY_LOGIN_PASSWORD | podman login -u $REGISTRY_LOGIN_USERNAME --password-stdin registry.suse.com
+          fi
+
```
Comment (dcermak) [inline .github/workflows/ci.yaml:335]:
```suggestion
        env:
          REGISTRY_LOGIN_USERNAME: ${{ secrets.REGISTRY_LOGIN_USERNAME }}
          REGISTRY_LOGIN_PASSWORD: ${{ secrets.REGISTRY_LOGIN_PASSWORD }}  }}
```

**Review by dcermak — COMMENTED** [review]
Could it be that the registry credential environment variables are miss-placed?

**Comment by dirkmueller** [comment]
> Could it be that the registry credential environment variables are miss-placed?

Yes, likely, included (I came up with that independent of your comment, which I only saw after updating the PR, my apologies)

File: pyproject.toml [hunk 3]
```diff
@@ -20,6 +20,9 @@ markers = [
     'bci-base_15.5',
     'bci-base_15.6',
     'bci-base_latest',
+    'bci-base-fips_15.3',
```
Comment (dcermak) [inline pyproject.toml:23]: no 15.4?

File: .github/workflows/ci.yaml [hunk 4]
```diff
@@ -319,6 +327,16 @@ jobs:
...
 
+      - name: Login to the registry if needed
+        run: |
+          if [ "$TARGET" = "ibs-released" ]; then
+            echo $REGISTRY_LOGIN_PASSWORD | podman login -u $REGISTRY_LOGIN_USERNAME --password-stdin registry.suse.com
+          fi
+        env:
+          REGISTRY_LOGIN_USERNAME: ${{ secrets.REGISTRY_LOGIN_USERNAME }}
+          REGISTRY_LOGIN_PASSWORD: ${{ secrets.REGISTRY_LOGIN_PASSWORD }}
+          TARGET: ${{ matrix.testing_target != '' && matrix.testing_target || 'obs' }}
```
Comment (dcermak) [inline .github/workflows/ci.yaml:338]: A bit simpler imho, but I don't insist on this change

```suggestion
      - name: Login to the registry if needed
        run: |
          echo $REGISTRY_LOGIN_PASSWORD | podman login -u $REGISTRY_LOGIN_USERNAME --password-stdin registry.suse.com
        if: ${{ matrix.testing_target == 'ibs_released' }}
        env:
          REGISTRY_LOGIN_USERNAME: ${{ secrets.REGISTRY_LOGIN_USERNAME }}
          REGISTRY_LOGIN_PASSWORD: ${{ secrets.REGISTRY_LOGIN_PASSWORD }}
```

File: bci_tester/data.py [hunk 5]
```diff
@@ -319,6 +324,23 @@ def create_BCI(
...
     )
+    if TARGET == "ibs-released":
+        if False:  # not yet released
+            LTSS_CONTAINERS.append(
+                create_BCI(
+                    build_tag=f"{APP_CONTAINER_PREFIX}/ltss/sle15.3/bci-base:{OS_CONTAINER_TAG}",
+                    available_versions=["15.3"],
+                    bci_type=ImageType.OS_LTSS,
+                )
+            )
```
Comment (dcermak) [inline bci_tester/data.py:335]: Do you have an ETA for the release?

**Review by dcermak — COMMENTED** [review]
I've added a few comments

File: pyproject.toml (hunk 3 above)
Comment (dirkmueller) [reply pyproject.toml:23]: 15.4 is not yet in LTSS

File: bci_tester/data.py (hunk 5 above)
Comment (dirkmueller) [reply bci_tester/data.py:335]: It is waiting for the BCI team according to maintenance. Release is ready. :shrug:

**Review by dcermak — COMMENTED** [review]
LGTM, but the tests currently fail with:
```
Trying to pull registry.suse.com/suse/ltss/sle15.3/bci-base-fips:15.3...
Error: initializing source docker://registry.suse.com/suse/ltss/sle15.3/bci-base-fips:15.3: reading manifest 15.3 in registry.suse.com/suse/ltss/sle15.3/bci-base-fips: requested access to the resource is denied
```

Also, this needs a rebase to get the latest helm & repo test fixes

**Comment by dirkmueller** [comment]
> LGTM, but the tests currently fail with:

yes, thats due to the "simplify login", it worked before. I've reverted that part for now. 

> Also, this needs a rebase to get the latest helm & repo test fixes

completed the rebase.

**Pull request merged**